app/jobs/validate_cocina_descriptive_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class ValidateCocinaDescriptiveJob < GenericJob
+  queue_as :default
+
+  ##
+  # A job that allows a user to verify that a spreadsheet for descriptive upload is valid
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [Hash] params additional parameters that an Argo job may need
+  # @option params [String] :csv_file csv data to validate
+  def perform(bulk_action_id, params)
+    super
+
+    csv = CSV.parse(params[:csv_file], headers: true)
+    with_csv_items(csv, name: 'Validate Cocina descriptive metadata') do |cocina_object, csv_row, success, failure|
+      DescriptionImport.import(csv_row: csv_row).either(
+        lambda { |description|
+          begin
+            cocina_object.new(description: description)
```

Comment:
You can replace this with `CocinaValidator.valdate(cocina_object, description: description`.  That can be chained with `bind` to the previous call, similar to https://github.com/sul-dlss/argo/blob/main/app/jobs/descriptive_metadata_import_job.rb#L21-L26

Reply:
Done.